code/css/theme-sky.css

```diff
@@ -0,0 +1,124 @@
+body {
+  background-color: #3498DB;
+}
+
+h1 {
+  text-align: center;
+  color: AliceBlue;
+}
+
+/* box for entire calculator - needed to position the content */
+
+.rcalc-container {
+  text-align: center;
+  margin-bottom: 200px;
+}
+
+/* calculator form */
+
+.rcalc-calculator {
+  background-color: #1c3968;
+  display: inline-block;
+  padding: 2px;
+}
+
+/* clear button */
+
+.rcalc-button-clear {
+  background-color: #681c1c;
+  border: none;
+  color: white;
+  font-size: 40px;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+}
+
+/* result display */
+
+#id-display-main {
```

Comment:
id don't use for styling ... if you need id attribute so it is not problem, but for css use class ... it is not problem have id and css in element.

Reply:
classes only used for styling, no ids